analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -108,10 +122,12 @@ private ProjectBuilder AddProject(AnalyzerLanguage language, string projectName,
             var project = Solution.AddProject(projectName, projectName, languageName);
 
             var compilationOptions = project.CompilationOptions.WithOutputKind(outputKind);
-            if (languageName == LanguageNames.CSharp)
+            compilationOptions = languageName switch
             {
-                compilationOptions = ((CSharpCompilationOptions)compilationOptions).WithAllowUnsafe(true);
-            }
+                LanguageNames.CSharp => ((CSharpCompilationOptions)compilationOptions).WithAllowUnsafe(true),
+                LanguageNames.VisualBasic => ((VisualBasicCompilationOptions)compilationOptions).WithGlobalImports(GlobalImport.Parse(DefaultGlobalImportsVisualBasic)),
```

Comment:
Maybe we can store the IEnumerable<GlobalImport> not to parse the imports for every project, since they don`t change between projects.